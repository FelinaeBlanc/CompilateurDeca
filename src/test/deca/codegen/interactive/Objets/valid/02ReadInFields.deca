class A {
    float x = readFloat(); 
    int y = readInt(); 

    float getX(){
        return x;  
    }
    
    int getY(){
        return y;  
    }

}

{
    A a = new A() ;
    println(a.getX()); 
    println(a.getY()); 
}