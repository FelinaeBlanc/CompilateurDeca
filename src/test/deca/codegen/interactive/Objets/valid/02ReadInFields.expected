1.25000e+01
14
